addons/clickable-quote-authors/addon.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "Clickable forum quote authors ",
+  "description": "Replaces author names in forum quotes with clickable links.",
+  "tags": ["community", "forums"],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["forums"]
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "userstyle.css",
+      "matches": ["forums"],
+      "if": {
+        "settings": {
+          "plainText": true
+        }
+      }
+    }
+  ],
+  "settings": [
+    {
+      "name": "Style as normal text",
+      "id": "plainText",
+      "type": "boolean",
+      "default": false
+    }
+  ],
+  "credits": [
+    {
+      "name": "just-a-hriday",
+      "link": "https://scratch.mit.edu/users/just-a-hriday"
+    }
+  ],
+  "enabledByDefault": false,
+  "updateUserstylesOnSettingsChange": true
```

```suggestion
  "dynamicEnable": true,
  "updateUserstylesOnSettingsChange": true
```
Dynamic disable will need some more work though, but that can be done later I guess.

---

But it doesn't enable dynamically.